defaults/config.js

```diff
@@ -35,6 +35,18 @@ module.exports = {
 	// This value is set to `9000` by default.
 	port: 9000,
 
+	// ### `socketPermissions`
+	//
+	// Set the permissions for the UNIX domain socket. Ignored when `host` is set
+	// to an IP address or hostname.
+	// When the value is not set (i.e. `undefined`) the permissions will not be
+	// changed and will default to whatever permissions the operating system uses.
+	//
+	// Example: Set this to `0o660` to allow read/write access for the owner and
+	//          the group and disallow access for anyone else.
+	//
+	socketPermissions: undefined,
```

Comment:
Can we be explicit the default value is undefined, similar to the other config options? Should we explicitly set it to 0o776 (or some other value, probably without execute)?

Reply:
I don't think setting it to a dummy value is something we should do... after all that makes some modes invalid.

I'd rather see undefined == respect umask

Reply:
in fact, do we enforce any perms at all? If not that could already be solved without an additional config option by setting the umask appropriately no?

Reply:
@brunnre8 I don't think the current default of 776 is correct, though (based on `Right now the socket is created with 0776 by default my machine.`)

Reply:
what's your umask? That permissions would imply o001 which is a very strange one to have